frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -177,17 +178,24 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
 
   getHistogram = (layerName: string, layer: DatasetLayerConfiguration) => {
     const { intensityRange } = layer;
+    const defaultIntensityRange = getDefaultIntensityRangeOfLayer(this.props.dataset, layerName);
+    const highestRangeValue = defaultIntensityRange[1];
     let histograms = [
-      { numberOfElements: 256, elementCounts: new Array(256).fill(0), min: 0, max: 255 },
+      {
+        numberOfElements: 5 * 256, // 0,
+        elementCounts: new Array(256).fill(5),
```

Comment:
Is that the same 5 or does it have a different semantics?

Reply:
I just tested out what happens if I change these two lines in a certain way (184 + 185). I accidentally committed this.

I'll revert this and let the default histogram contain no data (an empty array).